src/main/java/com/desiato/puresynth/services/AuthenticationService.java

```diff
@@ -26,13 +26,34 @@ public class AuthenticationService {
     public PureSynthToken authenticate(AuthenticationRequestDTO request) {
         log.info("Attempting to authenticate user with email: {}", request.email());
 
-        User user = findUserOrThrow(request);
+        User user;
+        try {
+            user = findUserOrThrow(request);
+            log.info("User found: {}", user.getEmail());
+        } catch (Exception e) {
+            log.error("Error finding user: {}", e.getMessage(), e);
+            throw e;
+        }
+
+        try {
```

Comment:
What's point of this?

Reply:
checking step-by-step all the parts all the methods in the authenticate request